turbopack/crates/turbopack-core/src/chunk/mod.rs

```diff
@@ -694,6 +694,10 @@ pub trait ChunkItem {
 
 #[turbo_tasks::value_trait]
 pub trait ChunkType: ValueToString {
+    /// Whether to perform vendor splitting for this chunk type.
+    /// TODO rethink this once we add a chunking strategy for prod (split chunks, etc.)
+    fn should_vendor_split(self: Vc<Self>) -> Vc<bool>;
```

Comment:
The name is a bit confusing. We do multiple level of splitting:

* vendor/app
* package name
* path

This function disables all of them, not only the vendor splitting. Maybe rename it to `should_split_by_heuristic`

Reply:
But we probably should rename it to `must_keep_item_order`, and (for now) implement that by only creating a single chunk. In future we might have a better implementation.